src/components/Calculator.js

```diff
@@ -1,48 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculator';
 
-export default class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-  }
-
-  render() {
-    const clickListener = (e) => {
-      const result = calculate(this.state, e.target.value);
-      this.setState(result);
-    };
-    const { total, next, operation } = this.state;
-    const result = `${total || ''}${operation || ''}${next || ''}`;
-    return (
-      <form action="#">
-        <div className="text">{result || '0'}</div>
-        <div className="calc-btn">
-          <input type="button" value="AC" onClick={clickListener} />
-          <input type="button" value="+/-" onClick={clickListener} />
-          <input type="button" value="%" onClick={clickListener} />
-          <input type="button" value="÷" onClick={clickListener} className="sign" />
-          <input type="button" value="7" onClick={clickListener} />
-          <input type="button" value="8" onClick={clickListener} />
-          <input type="button" value="9" onClick={clickListener} />
-          <input type="button" value="x" onClick={clickListener} className="sign" />
-          <input type="button" value="4" onClick={clickListener} />
-          <input type="button" value="5" onClick={clickListener} />
-          <input type="button" value="6" onClick={clickListener} />
-          <input type="button" value="-" onClick={clickListener} className="sign" />
-          <input type="button" value="1" onClick={clickListener} />
-          <input type="button" value="2" onClick={clickListener} />
-          <input type="button" value="3" onClick={clickListener} />
-          <input type="button" value="+" onClick={clickListener} className="sign" />
-          <input type="button" value="0" onClick={clickListener} className="zero" />
-          <input type="button" value="." onClick={clickListener} />
-          <input type="button" value="=" onClick={clickListener} className="sign" />
-        </div>
-      </form>
-    );
-  }
+export default function Calculator() {
```

Comment:
- [ ] [OPTIONAL] good job making the component functional, however I would advise you to use ES6 which includes the use of arrow functions instead of the `function` keyword

Reply:
Thank you @mikethreels for Approving my project
I have taken your advice of using ES6 which includes the use of arrow function and now my project is complete 

**Thank you!**